service/scaladsl/kafka/server/src/test/scala/com/lightbend/lagom/scaladsl/kafka/broker/ScaladslKafkaApiSpec.scala

```diff
@@ -82,12 +83,27 @@ class ScaladslKafkaApiSpec
 
   import application.materializer
 
-  private val kafkaServer = KafkaLocalServer(cleanOnStart = true)
+  private val kafkaServerClasspath: Seq[File] = TestBuildInfo.fullClasspath.toIndexedSeq
+  private var kafkaServer: Option[Process]    = None
 
   override def beforeAll(): Unit = {
     super.beforeAll()
 
-    kafkaServer.start()
+    kafkaServer = {
```

Comment:
Can we avoid this duplication? Also the port (`9092`) a few lines above.

Reply:
Yep.  Pushed to https://github.com/lagom/lagom/issues/2041.